### ISSUE TITLE
onAccessibilityTap not working for android

### ISSUE THREAD

**Issue by shannon-voda** [facebook/react-native#34136]
### Description

onAccessibilityTap on a View works on iOS but does not work on Android.

![image](https://user-images.githubusercontent.com/68305201/177455841-d1660217-9c1b-4522-8bfb-83903eb84e79.png)

Docs do not mention only working for iOS

### Version

0.66.2

### Output of `npx react-native info`

System:
    OS: macOS 12.4
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Memory: 1.88 GB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 15.14.0 - ~/.nvm/versions/node/v15.14.0/bin/node
    Yarn: 1.22.17 - /usr/local/bin/yarn
    npm: 7.7.6 - ~/.nvm/versions/node/v15.14.0/bin/npm
    Watchman: 2021.09.13.00 - /usr/local/bin/watchman
  Managers:
    CocoaPods: 1.11.2 - /usr/local/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.4, iOS 15.5, macOS 12.3, tvOS 15.4, watchOS 8.5
    Android SDK:
      API Levels: 29, 30, 31
      Build Tools: 29.0.2, 30.0.2, 30.0.3, 31.0.0
      System Images: android-21 | ARM EABI v7a, android-22 | Google APIs Intel x86 Atom, android-23 | Google APIs Intel x86 Atom, android-24 | Google APIs Intel x86 Atom, android-30 | Google APIs Intel x86 Atom, android-30 | Google Play Intel x86 Atom
      Android NDK: Not Found
  IDEs:
    Android Studio: 4.1 AI-201.8743.12.41.7199119
    Xcode: 13.4/13F17a - /usr/bin/xcodebuild
  Languages:
    Java: 1.8.0_321 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 17.0.2 => 17.0.2 
    react-native: 0.66.2 => 0.66.2 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found

### Steps to reproduce

Activate talkback
Tap on component with action
Double tap to activate action
Action will not activate

### Snack, code example, screenshot, or link to a repository

```
<View
        style={styles(this.context.theme).logoRow}
        accessible
        onAccessibilityTap={onPress}
        accessibilityRole='button'
        accessibilityHint={I18n.t('accessibility.dashboard.logoAndNameHint')}
      >
        <View style={{ flex: 0.3 }}>
          <Image style={styles(this.context.theme).logo} source={Images.redLogo} accessibilityIgnoresInvertColors />
        </View>
        {selectedServiceId && (
          <View style={{ flex: 0.7 }}>
            <MVAServiceName serviceId={selectedServiceId} onPress={onPress} />
          </View>
        )}
      </View>
```

**Comment by Droidking18** [comment]
Same issue here

**Comment by s-mirsky** [comment]
I also have this issue.

**Comment by shannon-voda** [comment]
Have recently updated to latest RN Version 0.69.3 and still encounter this issue.

**Comment by YanislavSpaceIQ** [comment]
I'm experiencing the same issue

**Comment by BasixKOR** [comment]
It seems like the property `onAccessibilityTap` is straight-up unimplemented on Android, as I couldn't any code referencing it on the Android code.

**Comment by Aksana-Tsishchanka** [comment]
I have the same issue with RN 0.72.6. Here is my code:

const a11yActions = [
    { name: 'add', label: 'Add the library' },
    { name: 'more', label: 'Open menu' },
  ]

<Pressable
      accessibilityActions={a11yActions}
      onAccessibilityTap={() => {
        setBg('yellow')
        if (event.nativeEvent.actionName === 'add') {
          firstButtonProps.onPress(event)
        }
        
        if (event.nativeEvent.actionName === 'more') {
          secondButtonProps?.onPress(event)
        }
      }}
    >
   {...}
</Pressable>

On Android when a screen reader is turn on, the bottom sheet with the defined actions appears but activation 'add' or 'more' does not call a provided callback.

Please reopen and fix the issue
